src/MumbleUDP.proto

```diff
@@ -68,10 +68,11 @@ message Ping {
 
 	// Below are the fields for the "additional information" that are filled out by the server on request.
 
-
-	// The version of the server encoded into a single integer. The 32 bits are partitioned in regions of (left to right)
-	// 16, 8 and 8 bits representing the major, minor and patch version number respectively.
-	uint32 server_version = 3;
+	// The version of the server in the legacy format.
+	uint32 server_version_v1 = 3;
+	// The version of the server in the new format.
+	// Necessary since patch level may exceed 255. (See https://github.com/mumble-voip/mumble/issues/5827)
+	uint64 server_version_v2 = 7;
```

Comment:
This is not needed, I think.
We can make a clear cut here given that we don't have to maintain backwards compatibility since 1.5 is not released yet.
We can simply define that for pings in UDP format, we will use the version scheme v2. End of the story. Should we ever introduce a version format v3, we will have to take measures such as this but for the time being, we don't have to

Reply:
We can still rename the field to `version_v2` to make it obvious and also add a comment stating that this is in version format 2, though